app/components/koi/tables/header.rb

```diff
@@ -4,8 +4,26 @@ module Koi
   module Tables
     module Header
       class NumberComponent < HeaderCellComponent
-        def default_html_attributes
-          { class: "number" }
+        def initialize(table, attribute, label: nil, link: {}, type: :number, width: "small", **html_attributes)
```

Comment:
why did you choose to go with "small", "medium" etc? Doesn't GovUK use `:s`, `:m`, `:l`?

In general, in Ruby if you're using a 'special' value, it should be a symbol instead of a string for faster comparisons.

Reply:
For reference, GovUk uses below,
- for `size`(eg. label) they use 'l', 'm', 's'. 
- for `width`(eg. text input) they use 'full', 'one-third' etc

I'll update this sizes to be of `:s`, `:m`, `:l`